backend/client-pkg/services/HelloService/index.js

```diff
@@ -0,0 +1,31 @@
+const http = require('http');
+
+const sayHello = (endpoint) => {
+  console.log('Sending hello!');
+  const request = http.request({
+    host: endpoint.host,
+    port: endpoint.port,
+    method: 'POST',
+    withCredentials: false,
+  }, (resp) => {
+    let respMsg;
+
+    resp.on('data', (dataChunk) => {
+      if (respMsg === undefined || respMsg === null) {
+        respMsg = dataChunk;
+      } else {
+        respMsg += dataChunk;
+      }
+    });
+
+    resp.on('end', () => {
+      console.log(respMsg);
```

Comment:
Why are we console.logging? Is this just a test? If it is just a test I think this should be pushed into a separate branch not the dev branch